vunit/sim_if/ghdl.py

```diff
@@ -108,6 +109,7 @@ def __init__(  # pylint: disable=too-many-arguments
         self._gtkwave_args = gtkwave_args
         self._backend = backend
         self._vhdl_standard = None
+        self._coverage_dirs = set()
```

Comment:
To avoid confusion in `merge_coverage`, where a lot of coverage dirs are thrown around, maybe rename this to `_coverage_test_dirs`

Reply:
Agreed

Reply:
Done